### PR TITLE
new: geany IDE

### PR DESCRIPTION
**Pull request by elieux**

### CODE REVIEW

File: mingw-w64-geany/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+depends=("${MINGW_PACKAGE_PREFIX}-gtk2" 
+         "${MINGW_PACKAGE_PREFIX}-hicolor-icon-theme")
+makedepends=('perlxml'
+             'intltool'
+             "${MINGW_PACKAGE_PREFIX}-gcc")
+install=geany-${CARCH}.install
+optdepends=(
+            #'geany-plugins: various extra features'
+            #'vte: terminal support'
+            'python2')
```
Comment (Alexpux) [inline mingw-w64-geany/PKGBUILD]: ${MINGW_PACKAGE_PREFIX}-python2

File: mingw-w64-geany/0003-Use-FHS.patch [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+index 0d952bc..fc5ca18 100644
+--- a/src/main.c
++++ b/src/main.c
+@@ -414,8 +414,8 @@ static void setup_paths(void)
+ 	 * documentation and data files */
+ 	gchar *install_dir = win32_get_installation_dir();
+ 
+-	data_dir = g_build_filename(install_dir, "data", NULL); /* e.g. C:\Program Files\geany\data */
+-	doc_dir = g_build_filename(install_dir, "doc", NULL);
++	data_dir = g_build_filename(install_dir, "share", "geany", NULL); /* e.g. C:\Program Files\geany\data */
```
Comment (Alexpux) [inline mingw-w64-geany/0003-Use-FHS.patch]: I think you are wrong here. Need to be:
````
data_dir = g_build_filename(install_dir, "share", "geany", NULL); /* e.g. C:\Program Files\\share\geany */
````

File: mingw-w64-geany/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  patch -p1 -i "${srcdir}/0002-Fix-plugin-compilation.patch"
+  patch -p1 -i "${srcdir}/0003-Use-FHS.patch"
+}
+
+build() {
+  mkdir -p "build-${MINGW_CHOST}"
+  cd "build-${MINGW_CHOST}"
+
+  # libiberty
+  export LIBRARY_PATH="${MINGW_PREFIX}/lib/binutils"
```
Comment (Alexpux) [inline mingw-w64-geany/PKGBUILD]: maybe do like:
````
LDFLAGS+=" -L${MINGW_PREFIX}/lib/binutils"
````

File: mingw-w64-geany/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+  # config.h
```
Comment (Alexpux) [inline mingw-w64-geany/PKGBUILD]: Add to CPPFLAGS

File: mingw-w64-geany/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  export C_INCLUDE_PATH="$(pwd)"
```
Comment (Alexpux) [inline mingw-w64-geany/PKGBUILD]: Add to CPPFLAGS

File: mingw-w64-geany/0003-Use-FHS.patch (hunk 2 above)
Comment (elieux) [inline mingw-w64-geany/0003-Use-FHS.patch]: I didn't change the comment at all. I will remove it to avoid confusion.

**Comment by elieux** [comment]
I addressed all your comments.

**Comment by mingwandroid** [comment]
Thanks David.

**Pull request merged**